Cargo.toml

```diff
@@ -13,7 +13,14 @@ categories = ["api-bindings", "os::windows-apis"]
 
 [dependencies]
 cfg-if = "1.0"
-winapi = { version = "0.3.9", features = ["impl-default", "impl-debug", "minwindef", "minwinbase", "timezoneapi", "winerror", "winnt", "winreg", "handleapi"] }
+windows-sys = {version = "0.45.0", features = [
```

Comment:
`windows-sys` 0.48 was released some days ago.

Reply:
You're right. I was relying on my IDE's autocomplete, which appears to be outdated. I've bumped the version number up. Tests still pass.